lib/views/widgets/tabs/tab_buttons.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:sun_flutter_capstone/consts/global_style.dart';
+
+class TabButtons extends StatelessWidget {
+  final TabController tabController;
+  final int index;
+
+  const TabButtons({
+    Key? key,
+    required this.tabController,
+    required this.index,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: 50,
+      decoration: BoxDecoration(
+        color: AppColor.light,
+        borderRadius: BorderRadius.circular(
+          25.0,
+        ),
+      ),
+      child: Expanded(
+        child: Padding(
+          padding: const EdgeInsets.all(5),
+          child: TabBar(
+            controller: tabController,
+            indicator: BoxDecoration(
+              borderRadius: BorderRadius.circular(
+                25.0,
+              ),
+              color: index == 0 ? AppColor.pink : AppColor.darkBlue,
+            ),
+            unselectedLabelColor: Colors.black,
+            labelColor: Colors.white,
+            tabs: const [
+              Tab(
+                child: Text('Add Expense'),
```

Comment:
Add semi-bold style for the font based sa figma, it's semi bold

```suggestion
                child: Text('Add Expense', style: TextStyle(fontWeight: FontWeight.w600)),
```

Reply:
Will do, miss.. I just noticed.. the color is also different diay 😅